Blish HUD/_Utils/DrawUtil.cs

```diff
@@ -47,7 +47,7 @@ private static string WrapTextSegment(BitmapFont spriteFont, string text, float
             string[] words      = text.Split(' ');
             var      sb         = new StringBuilder();
             float    lineWidth  = 0f;
-            float    spaceWidth = spriteFont.MeasureString(" ").Width;
+            float    spaceWidth = spriteFont.MeasureString("_").Width;
```

Comment:
Perhaps a more conservative value like 4 or 5 can be used because now I see the text wraps too early and it leaves a lot of empty space at the end of a line.

```diff
-float spaceWidth = spriteFont.MeasureString("_").Width;
+float spaceWidth = 4;
```